docs/examples/kfserving/README.ipynb

```diff
@@ -45,12 +45,35 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.                "runtime": "tempo.kfserving.KFServingKubernetesRuntime",
</pre><p>In this case you would probably be using <code>from tempo.serve.metadata import KFServingOptions</code>  so you don't ahve to specify the underlying runtime in <code>remote_options</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/tempo/pull/178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>